returnn/datasets/meta.py

```diff
@@ -1406,13 +1407,19 @@ def __init__(
           Now it could happen that ceildiv(data_len1 + data_len2, 6) < align_len1 + align_len2.
           This option would repeat intermediate ending frames such that data_len1 % 6 == 0,
           by setting it to {"data": 6}.
+        :param dict[str,(str,int)]|None pad_truncate_data_to_multiple_of_target_len: data_key -> (target_key, multiple).
+            Force data_len == target_len * multiple for specified data and target.
+            Similar to repeat_in_between_last_frame_up_to_multiple_of but also truncates the data.
+            Mainly for raw wav alignment, where data_len could be slightly longer than target_len * multiple
+            due to mismatch in feature extraction.
         :param bool use_cache_manager:
         :param dict[(int,int),dict] epoch_wise_filter: see :class:`EpochWiseFilter`
         """
         super(ConcatSeqsDataset, self).__init__(**kwargs)
         self.seq_tag_delim = seq_tag_delim
         self.remove_in_between_postfix = remove_in_between_postfix or {}
         self.repeat_in_between_last_frame_up_to_multiple_of = repeat_in_between_last_frame_up_to_multiple_of or {}
+        self.pad_truncate_data_to_multiple_of_target_len = pad_truncate_data_to_multiple_of_target_len or {}
```

Comment:
I think this should have an assertion that not both properties are set?

Reply:
There is no unexpected behavior if both parameters are set(but probably redundant), so I don't know whether that would make sense.

Reply:
Yea, for me, also not strictly necessary.
(@NeoLegends, can you re-review?)